src/store/getters/state-getters.ts

```diff
@@ -563,7 +563,12 @@ export const hasPeopleAndRolesChanged = (state: StateIF): boolean => {
   let currentOrgPersons = getPeopleAndRoles(state)
   let originalOrgPersons = getEntitySnapshot(state)?.orgPersons
 
-  return (!isSame(currentOrgPersons, originalOrgPersons, ['action']))
+  return (!isSame(currentOrgPersons, originalOrgPersons, ['action', 'confirmNameChange']))
+}
+
+/** Is true when the minimum partners met. */
+export const hasMinimumPartners = (state: StateIF): boolean => {
+  return getPeopleAndRoles(state).filter(party => !party.action?.includes(ActionTypes.REMOVED)).length >= 2
```

Comment:
Looks fine but could you add a comment explaining what is being computed here? (I understand this today because I'm reviewing this PR, but by next week I may forget why you're excluding parties with the REMOVED action,)

Reply:
Great point, will add!

Reply:
Added!